### ISSUE TITLE
[WCAG] Ensure Focus Indicators Are Consistently Styled

### ISSUE THREAD

**Issue by kenchase** [Canadian-Geospatial-Platform/geoview#3236]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

Focused elements should have the same styles (border size and colour, background colour, etc.)

For example, the layer items (left panel), in the Details panel and the Layers panel do not have the same styles.

Also,

In style.css, the following rule seems heavy-handed. It might need to be, but it needs to be reviewed.

[class*='mapContainer'].keyboard-focused {
border: #000 solid 2px !important;
}

Search the codebase for ":focus-visible" to find many overrides.

For example, it adds a black border to input fields (on keyboard focus), which are not necessarily needed (see the Geolocator search input, for example. It uses CSS to hide the black border.)

Maybe the rule can be better targeted so that overrides can be minimized?
